cli/types/tests/cypress-tests.ts

```diff
@@ -98,6 +98,10 @@ cy.wrap({ foo: [1, 2, 3] })
     s
   })
 
+cy.wrap(['foo', 'bar']).its(1)
```

Comment:
// can you add expected type or value assertion here?

Reply:
done